### DISCUSSION TITLE
About blocking page elements with UBlock Origin NSFW

### DISCUSSION THREAD

**Discussion by BrunoBonas** [uBlockOrigin/uAssets#24848] [Q&A]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [X] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
NSFW
https://discord.com/channels/966183700272398346/1039377295619674134 NSFW
```

### Description

Hello, how are you?

 THANK YOU to the creator of this program , because it has the tool I was looking for to block certain content that harms me , but the problem is that they are not working on certain pages and giving error.
An example is when I use the "lightning bolt" as in "figure 1 and 2" it works once or twice, but when I use it a "3" time it gives a problem on the page and it tells you to restart as in figure 3. and it comes back all over again as in "figure 4" . What could this error be and how can I solve it?

Thanks for reading, sorry for the long text and thanks for answering if possible and congratulations on the program which is 10 out of 10.
 ! https://github.com/uBlockOrigin/uAssets/issues/2
    ||data.inertanceretinallaurel.com^

### Other extensions used

Deepl Translator
Browsec

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![1](https://github.com/user-attachments/assets/c3a18124-44e1-4b8c-8c3b-c317538df588)
![2](https://github.com/user-attachments/assets/d0e62d9b-ed79-4c0d-b022-63eb64666d52)
![3](https://github.com/user-attachments/assets/f86161be-43e1-4e3f-bd28-7a39738a9eaa)
![4](https://github.com/user-attachments/assets/ac4fd855-d7d6-44bf-917a-9a570fcb941d)
![5](https://github.com/user-attachments/assets/adfd544b-c876-498a-8b31-6882b3472578)

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml
uBlock Origin: 1.59.0
Chromium: 127
filterset (summary):
 network: 154012
 cosmetic: 102580
 scriptlet: 29918
 html: 0
listset (total-discarded, last-updated):
 removed:
  spa-1: null
 added:
  adguard-popup-overlays: 27666-820, 16m
  adguard-mobile-app-banners: 5391-9, 16m
  adguard-other-annoyances: 14860-326, 16m
  adguard-widgets: 2627-19, 16m
  easylist-annoyances: 4684-50, 16m
  easylist-chat: 205-0, 16m
  easylist-newsletters: 7484-24, 16m
  easylist-notifications: 3125-1191, 16m
  ublock-annoyances: 5983-158, 42m Δ
 default:
  user-filters: 5-0, never
  ublock-filters: 39570-591, 42m Δ
  ublock-badware: 10217-1, 42m Δ
  ublock-privacy: 1044-2, 42m Δ
  ublock-unbreak: 2468-2, 42m Δ
  ublock-quick-fixes: 138-0, 42m Δ
  easylist: 87842-630, 42m Δ
  easyprivacy: 53073-770, 42m Δ
  urlhaus-1: 21325-0, 16m
  plowe-0: 3549-0, 16m
filterset (user): [array of 5 redacted]
trustedset:
 added: [array of 3 redacted]
switchRuleset:
 added: [array of 2 redacted]
userSettings:
 advancedUserEnabled: true
 userFiltersTrusted: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1352 ms (selfie)
 maxAssetCacheWait: 254 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 0
```
</details>

**Comment by MasterKia** [comment]
Use the element picker (next to the 'lightning' icon).

**Comment by BrunoBonas** [comment]
> Use the element picker (next to the 'lightning' icon).

So I tried using the "element picker" like in "figure 1, 2 and 3", but when I do that the page goes crazy and doesn't let anything else appear on the page.

How do I fix this?

![1](https://github.com/user-attachments/assets/8cb803f0-50a7-4fac-aabc-4d0f93c86a88)
![2](https://github.com/user-attachments/assets/224f0895-e802-4941-9d68-6daaa3943740)
![3](https://github.com/user-attachments/assets/e767b4ef-44ac-41fa-839a-263b25325144)

```yaml
uBlock Origin: 1.59.0
Chromium: 127
filterset (summary):
 network: 160274
 cosmetic: 102625
 scriptlet: 29925
 html: 0
listset (total-discarded, last-updated):
 added:
  adguard-popup-overlays: 27666-119, 2d.23h.12m
  adguard-mobile-app-banners: 5391-9, 2d.23h.12m
  adguard-other-annoyances: 14860-642, 2d.23h.12m
  adguard-widgets: 2627-26, 2d.23h.12m
  easylist-annoyances: 4684-194, 2d.23h.12m
  easylist-chat: 205-0, 2d.23h.12m
  easylist-newsletters: 7484-531, 2d.23h.12m
  easylist-notifications: 3125-1347, 2d.23h.12m
  ublock-annoyances: 5975-120, 1h.28m Δ
 default:
  user-filters: 9-0, never
  ublock-filters: 39618-355, 1h.28m Δ
  ublock-badware: 10443-6, 1h.28m Δ
  ublock-privacy: 1044-2, 1h.28m Δ
  ublock-unbreak: 2470-0, 1h.28m Δ
  ublock-quick-fixes: 129-0, 1h.28m Δ
  easylist: 87971-608, 1h.28m Δ
  easyprivacy: 53076-22, 1h.28m Δ
  urlhaus-1: 27255-0, 22h.34m
  plowe-0: 3549-621, 2d.23h.12m
filterset (user): [array of 9 redacted]
trustedset:
 added: [array of 7 redacted]
switchRuleset:
 added: [array of 3 redacted]
userSettings:
 advancedUserEnabled: true
 userFiltersTrusted: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 3355 ms
 maxAssetCacheWait: 1824 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 0
 extended:
  ##li.mainCard_a6d69a.card_a6d69a.container_a57509:nth-of-type(6)
  ##li.mainCard_a6d69a.card_a6d69a.container_a57509:nth-of-type(4)
  ##li.mainCard_a6d69a.card_a6d69a.container_a57509:nth-of-type(5)
  ##li.mainCard_a6d69a.card_a6d69a.container_a57509:nth-of-type(3)
```

**Reply by D4niloMR** [reply]
What exactly are you trying to do?
